code/datums/emergency_calls/xeno_cultists.dm

```diff
@@ -0,0 +1,37 @@
+//Xeno Cultists
+/datum/emergency_call/xeno_cult
+	name = "Xeno Cultists"
+	mob_max = 10
```

Comment:
This should be the more standard "squad" at six mobs. I'd recommend two if you want a bigger one for more generalized event stuff.